### ISSUE TITLE
Modified po files in working repository

### ISSUE THREAD

**Issue by elemoine** [Geoportail-Luxembourg/geoportailv3#523]
Git sees modified po files again in my working repository:

```
        modified:   geoportailv3/locale/de/LC_MESSAGES/geoportailv3-server.po
        modified:   geoportailv3/locale/en/LC_MESSAGES/geoportailv3-server.po
        modified:   geoportailv3/locale/fr/LC_MESSAGES/geoportailv3-server.po
        modified:   geoportailv3/locale/lb/LC_MESSAGES/geoportailv3-server.po
```

Does someone know what the problem is?

**Comment by sbrunner** [comment]
Changing the sources ?

**Comment by sbrunner** [comment]
Fixed by #565
